src/ORM/Connect/Query.php

```diff
@@ -3,12 +3,18 @@
 namespace SilverStripe\ORM\Connect;
 
 use SilverStripe\Core\Convert;
-use Iterator;
 use Traversable;
 
 /**
- * Abstract query-result class. A query result provides an iterator that returns a map for each record of a query
- * result.
+ * Abstract query-result class. A query result provides an IteratorAggregate that returns a map for each record of a
+ * query result.
+ *
+ * This should be subclassed by an actual database implementation. It will only
+ * ever be constructed by a subclass of Database. The result of a database query - an iteratable object
```

Comment:
```suggestion
 * ever be constructed by a subclass of Database. The result of a database query - an iterable object
```
As in https://www.php.net/manual/en/language.types.iterable.php

Reply:
Done